scss/civicrm/contact/_tabs.scss

```diff
@@ -1,194 +1,195 @@
-/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+/* stylelint-disable selector-max-id */
 
-.crm-contact-page {
-  padding: 4px;
+#mainTabContainer {
+  background-color: transparent;
+  display: flex;
   position: relative;
 
-  #mainTabContainer {
-    background-color: transparent;
-    display: flex;
-    padding: 0;
+  #ui-id-11 {
+    overflow: auto;
 
-    // This snippet is used to hide tab panel until javascript loads tabs properly and generates required classes.
-    &:not(.ui-tabs) {
-      .ui-tabs-panel {
-        visibility: hidden;
-      }
+    .crm-accordion-body,
+    .contact-activity-selector-activity {
+      overflow: auto;
     }
+  }
+
+  .ui-dialog {
+    background: $crm-white;
+  }
 
-    .crm-contact-tabs-list {
-      background: $crm-white;
-      border-radius: 0;
-      box-shadow: $box-shadow;
-      flex-shrink: 0;
-      margin: 0;
-      padding: 0;
-      width: 200px;
+  // override of jquery-ui.css that creates weird white background
```

Comment:
Let's remove any opinion based statements from comments and stay cold-blooded:

`// Override a white background rule made by jquery-ui.css`